tests/stub/server_side_routing/test_server_side_routing.py

```diff
@@ -70,14 +50,22 @@ def test_direct_connection_without_url_params(self):
     def test_direct_connection_with_url_params(self):
         params = "region=china&policy=my_policy"
         uri = "bolt://%s?%s" % (self._server.address, params)
-        self._server.start(script=direct_connection_without_routing_ssr_script,
-                           vars={
-                               "#VERSION#": "4.1"
-                           })
+        self._start_server()
         try:
             driver = Driver(self._backend, uri, self._auth, self._userAgent)
-        except Exception:
+        except types.DriverError:
             pass
+        except types.BackendError:
+            if get_driver_name() in ["javascript"]:
+                # TODO: this shouldn't be communicated as backend error
```

Comment:
Sure it's an issue with the backend, it's not capturing correctly the exception.